addons/website_sale_slides/views/website_slides_templates.xml

```diff
@@ -31,4 +42,16 @@
     </xpath>
 </template>
 
+<template id="buy_now_course"
```

Comment:
Side note: indentation seems strange here. We could also update the id to something like course_sidebar_buy_now_course or something like that to be more specific.